src/components/navigation/Navigation.tsx

```diff
@@ -43,52 +57,111 @@ const Navigation = ({ onNavigationToggle, open, width }: Props) => {
-                        link={routeDetails.collections.path}
-                    />
-                    <ListItemLink
-                        icon={<StorageIcon sx={{ color: '#F6FAFF' }} />}
-                        title={routeDetails.materializations.title}
-                        link={routeDetails.materializations.path}
-                    />
-                    <ListItemLink
-                        icon={<CableIcon sx={{ color: '#F6FAFF' }} />}
-                        title={routeDetails.connectors.title}
-                        link={routeDetails.connectors.path}
-                    />
-                    <ListItemLink
-                        icon={
-                            <AdminPanelSettingsIcon sx={{ color: '#F6FAFF' }} />
-                        }
-                        title={routeDetails.admin.title}
-                        link={routeDetails.admin.path}
-                    />
-                </List>
+            <Box
+                sx={{
+                    height: '100%',
+                    display: 'flex',
+                    flexDirection: 'column',
+                    justifyContent: 'space-between',
+                    overflowX: 'hidden',
+                }}
+            >
+                <Box>
+                    <Box sx={{ mb: 16, pl: 1 }}>
+                        <Box
+                            sx={{
+                                pt: 1,
+                                pb: 0.25,
+                                display: 'flex',
+                                flexGrow: 1,
+                                alignItems: 'center',
+                            }}
+                        >
+                            <IconButton
+                                aria-label={intl.formatMessage({
+                                    id: 'header.openNavigation.ariaLabel',
+                                })}
+                                onClick={openNavigation}
+                                sx={{
+                                    display: 'inline-flex',
+                                    justifyContent: 'left',
+                                    flexShrink: 0,
+                                }}
+                            >
+                                <MenuIcon />
+                            </IconButton>
+
+                            <Typography
+                                sx={{ width: 136, ml: 2, flexShrink: 0 }}
+                            >
+                                Main Menu
+                            </Typography>
+                        </Box>
+
+                        <UserMenu />
+                    </Box>
+
+                    <List
+                        aria-label={intl.formatMessage({
+                            id: 'navigation.ariaLabel',
+                        })}
+                    >
+                        <ListItemLink
+                            icon={<InputIcon sx={{ color: iconColor }} />}
```

Comment:
If you're already storing the `iconColor` in a variable you can just store the entire `sx` property in one and reference it